eng/tox/tox.ini

```diff
@@ -51,27 +75,25 @@ ignore_args=--ignore=.tox --ignore=build --ignore=.eggs --ignore=samples
 default_args = -rsfE --junitxml={tox_root}/test-junit-{envname}.xml --verbose --cov-branch --durations=10 --ignore=azure {[pytest]ignore_args} --log-cli-level={pytest_log_level}
 
 [testenv]
+uv_seed = true
 parallel_show_output =True
 skip_install = true
 skipsdist = true
 usedevelop = false
 passenv = *
 download=true
+# Allow both pip and uv as external commands since we support both via TOX_PIP_IMPL
+allowlist_externals = uv,pip
 requires=
   {[packaging]pkgs}
 setenv =
   SPHINX_APIDOC_OPTIONS=members,undoc-members,inherited-members
-  PIP_EXTRA_INDEX_URL=https://pypi.python.org/simple
+  UV_EXTRA_INDEX_URL=https://pypi.python.org/simple
```

Comment:
Removing default configuration elements like this without replacing shouldn't be done. speaking specifically about"

`VIRTUALENV_*` and `PIP_EXTRA_INDEX_URL` exists making it set by default.